src/main/kotlin/com/jetbrains/snakecharm/lang/parser/SnakemakeStatementParsing.kt

```diff
@@ -29,6 +30,39 @@ class SnakemakeStatementParsing(
         futureFlag: FUTURE?
 ) : StatementParsing(context, futureFlag) {
 
+    private val keywordToParsingFunction = ImmutableMap.Builder<String, (IElementType?) -> Unit>()
```

Comment:
Why do we need this change if you remap identifier token to snakemake specific token type? Could we leave source code after remap as it was before (based on token types comparison, not on matching identifier text content)?